static/js/drafts.js

```diff
@@ -428,6 +428,28 @@ function update_rendered_drafts(has_drafts_from_conversation, has_other_drafts)
     }
 }
 
+function updateButtonPlacement() {
```

Comment:
It would also be great if this could be handled by flexbox or something similar, instead of manual pixel adjustments here.

Reply:
this has to be changed after #21280 get merged anyway, so in my opinion changing this so it uses flexbox seems like a lot of unnecessary work, when it will have to be changed anyway
@alya what do you think?

Reply:
Can you explain more about how it will need to be changed? I'd ideally review this PR without it having the manual pixel adjustments, but maybe that means building it over that other PR?